pkg/datapath/manager.go

```diff
@@ -95,3 +95,16 @@ func (m *Manager) GetAsyncBR(jobName string) AsyncBR {
 		return nil
 	}
 }
+
+// CanAcceptNewTask checks if a new task can be accepted based on the concurrent limit.
+// This is a lightweight check that doesn't create any resources.
+func (m *Manager) CanAcceptNewTask(resume bool) bool {
```

Comment:
`resume` is not used and for long term, it doesn't look necessary either, because resuming tasks have high priority. Therefore, let's remove this `resume` parameter.